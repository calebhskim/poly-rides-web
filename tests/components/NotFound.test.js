import React from 'react';

import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

import NotFound from '../../src/components/NotFound';

describe('<NotFound />', () => {
  it('renders properly', () => {
    const wrapper = shallow(<NotFound />);

    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });
});
