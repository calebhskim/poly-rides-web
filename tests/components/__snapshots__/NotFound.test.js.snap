// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFound /> renders properly 1`] = `
<View>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
  >
    404 not found
  </Text>
</View>
`;
